python/mxnet/contrib/onnx/_import/translation_utils.py

```diff
@@ -148,21 +152,29 @@ def _fix_bias(op_name, attrs, num_inputs):
         raise ValueError("Unexpected number of inputs for: {}".format(op_name))
     return attrs
 
-def _fix_bias_shape(op_name, inputs, cls):
+def _fix_broadcast(op_name, inputs, broadcast_axis, cls):
```

Comment:
Shouldn't this be `obj` instead of `cls`?  Same question for everywhere cls is used in `op_translations` and `translation_utils`

Reply:
no I meant to use ``cls``, its just a convention, but the same way ``self`` is used to access an attribute inside the object (class) itself.``cls`` is often used to reference class and instance variables outside the object.

Reply:
Using `cls` for an instance is misleading. `cls` is the preferred variable name for anything that is meant to be class. For your case, since it is an instance of a class that is passed to `_fix_broadcast` and elsewhere, it should be something that indicates an instance like `obj`.

Reply:
okay, i will rename it to ``proto_obj``